### PR TITLE
Ocrvs 2564: NID regex pattern configuration enabled for national system admin. 

### PR DESCRIPTION
**Pull request by emon-swe-sust**
nid_Pattern

This PR has an associate PR on farazaland (https://github.com/opencrvs/opencrvs-farajaland/pull/196)

### CODE REVIEW

File: packages/client/src/views/SysAdmin/Config/Application.tsx [hunk 1]
```diff
@@ -130,13 +132,18 @@ function GeneralTabContent({
...
         },
         {
-          label: intl.formatMessage(messages.uniqueIdentificationNumberLabel),
-          value:
-            offlineCountryConfiguration.config.NID_NUMBER_PATTERN.pattern.toString(),
+          label: intl.formatMessage(messages.nidPatternTitle),
+          value: (
+            <div id="nidPattern_value_container">
+              {offlineCountryConfiguration.config.NID_NUMBER_PATTERN.toString()}
+            </div>
```
[the remark this answers is not in the document]
Comment (emon-swe-sust) [reply packages/client/src/views/SysAdmin/Config/Application.tsx:139]: no, I changed it !

File: packages/components/src/components/icons/Cross.tsx [hunk 2]
```diff
@@ -20,6 +20,9 @@ export const Cross = (props: React.HTMLAttributes<SVGElement>) => {
     case 'currentColor':
       fill = 'currentColor'
       break
+    case 'red':
+      fill = '#D53F3F'
```
Comment (Zangetsu101) [inline packages/components/src/components/icons/Cross.tsx:24]: Maybe we can use the color from `colors.ts`?

File: packages/client/src/utils/validate.ts [hunk 3]
```diff
@@ -550,10 +549,7 @@ export const validIDNumber =
...
 
       return {
-        message: messages.validNationalId,
-        props: {
-          validLength: num
-        }
+        message: messages.validNationalId
       }
     }
     return undefined
```
Comment (atiqzaman-dsi) [inline packages/client/src/utils/validate.ts:555]: `duplicateIDNumber` will have to be changed.
`if (value && valueToCheck && +value === +valueToCheck) ` Change this line.

**Review by atiqzaman-dsi — CHANGES_REQUESTED** [review]

File: packages/client/src/utils/validate.ts (hunk 3 above)
Comment (atiqzaman-dsi) [reply packages/client/src/utils/validate.ts:555]: This code can be found in latest `develop` branch. @emon-swe-sust please test the `duplicateIDNumber` in `correct record`.

**Pull request closed without merging**